packages/nodes-base/nodes/Onfleet/descriptions/DestinationDescription.ts

```diff
@@ -27,22 +27,22 @@ export const destinationOperations = [
 		],
 		default: 'get',
 	},
-] as INodeProperties[];
+];
 
 const unparsedField = {
-	displayName: 'Unparsed adress',
+	displayName: 'Unparsed Address',
 	name: 'unparsed',
 	type: 'boolean',
 	description: 'Whether the address is specified in a single',
```

Comment:
Unfinished sentence here: `Whether or not the address is specified in a single unparsed string`